cob_bringup/robots/cob4-3.xml

```diff
@@ -31,6 +31,10 @@
 		<include file="$(find cob_bringup)/controllers/base_controller_plugin.launch">
 			<arg name="robot" value="$(arg robot)"/>
 		</include>
+		<include file="$(find cob_omni_drive_controller)/launch/stuck_detector.launch">
```

Comment:
I wasn't aware that cob_omni_drive_controller was pulled in via cob_hardware_config. Made it explicit now.